TombEngine/Specific/winmain.cpp

```diff
@@ -393,7 +393,7 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine
 	Sound_Init();
 
 	// Initialise input
-	InitialiseInput(App.WindowHandle);
```

Comment:
We use `Initialise` everywhere. Either we rename everything to `Initialize` (which I think must be done, but in other PR), or let's stick to `Initialise` for now.

Reply:
Sure, I will make another PR for it.